### PR TITLE
initial commit

### PR DESCRIPTION
**Pull request by AleksandrPok**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -19,11 +20,29 @@
...
+            <artifactId>junit</artifactId>
+            <version>4.12</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.junit.jupiter</groupId>
+            <artifactId>junit-jupiter-api</artifactId>
+            <version>5.7.0</version>
+            <scope>test</scope>
+        </dependency>
```
Comment (vicire) [inline pom.xml]: do we need junit dependency here?

File: pom.xml [hunk 2]
```diff
@@ -19,11 +20,29 @@
...
         <dependency>
             <groupId>mysql</groupId>
             <artifactId>mysql-connector-java</artifactId>
             <version>8.0.22</version>
         </dependency>
+        <dependency>
+            <groupId>javax.servlet</groupId>
+            <artifactId>javax.servlet-api</artifactId>
+            <version>3.1.0</version>
```
Comment (vicire) [inline pom.xml]: we need to to use alfa version of dependencies

File: pom.xml [hunk 3]
```diff
@@ -40,6 +59,7 @@
                     </execution>
                 </executions>
                 <configuration>
+
```
Comment (vicire) [inline pom.xml]: remove empty line

File: src/main/java/taxi/dao/DriverDao.java [hunk 4]
```diff
@@ -0,0 +1,7 @@
+package taxi.dao;
+
+import taxi.model.Driver;
+
+public interface DriverDao extends GenericDao<Driver, Long> {
+
```
Comment (vicire) [inline src/main/java/taxi/dao/DriverDao.java]: remove empty line

**Review by vicire — CHANGES_REQUESTED** [review]
You need to implement GenericService interface

**Review by vicire — APPROVED** [review]

**Review by KseniiaMakarova — APPROVED** [review]

**Pull request merged**